app/Http/Controllers/CityController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\City;
+use App\Models\Area;
+use Illuminate\Http\Request;
+
+class CityController extends Controller
+{
+
+    public function index()
+    {
+        $city = City::all();
+        return response()->json(["That are all cities",$city]);
```

Comment:
Put the value of your message in a key for easy referencing, refactor this across your controllers and add proper status codes as well.
Example: response()->json(["message" => "That are all cities", "data" => $city], 200);